ts/modules/full-screen.src.ts

```diff
@@ -18,63 +15,169 @@ import H from '../parts/Globals.js';
  */
 declare global {
     namespace Highcharts {
+        interface Chart {
+            browserProps: {
+                fullscreenChange: string;
+                requestFullscreen: string;
```

Comment:
Here you should set requestFullscreen type to the literal string types, that are possible: `requestFullscreen: ('msRequestFullscreen'|'mozRequestFullscreen'|'requestFullscreen'|'webkitRequestFullscreen')`